app/decorators/service_decorator.rb

```diff
@@ -56,4 +56,30 @@ def as_json(options = {})
   def backend_api?
     false
   end
+
+  def links
+    [
+      { name: 'Edit', path: '' },
+      { name: 'Overview', path: h.admin_service_path(object) },
+      { name: 'Analytics', path: h.admin_service_stats_usage_path(object) },
+      { name: 'Applications', path: h.admin_service_applications_path(object) },
+      { name: 'ActiveDocs', path: h.admin_service_api_docs_path(object) },
+      { name: 'Integration', path: h.path_to_service(object) },
+    ]
+  end
+
+  def apps_count
+    cinstances.size
```

Comment:
maybe a `sum` of all the `contracts_count` field in the DB for all the `application_plans` of this service.
that would be more efficient.

Reply:
similarly to https://github.com/3scale/porta/pull/2194/files#r504635406, next round for improvements 👍